ReSharper.FSharp/src/FSharp.Psi.Features/src/FSharpTypingAssist.fs

```diff
@@ -1493,6 +1495,47 @@ type FSharpTypingAssist(lifetime, solution, settingsStore, cachingLexerService,
 
         insertCharInBrackets context lexer atChars typedQuotationBrackets LeftBracketOnly.No
 
+    member x.HandleXmlDocTag(context: ITypingContext) =
+        let textControl = context.TextControl
+        let offset = textControl.Caret.Offset()
+        let mutable lexer = Unchecked.defaultof<_>
+        if not (x.GetCachingLexer(textControl, &lexer)) then false else
+
+        if offset < 4 then false else
```

Comment:
I wonder if we should also support typing `<` right after `///` and adding the missing space?
```fsharp
///{caret}
let _ = ...
```

Reply:
there is a small bug here; we should, of course, check it for < 3